target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -256,7 +286,7 @@ struct airoha_pinctrl {
 
 	struct mutex mutex;
 	struct {
-		void __iomem *mux[3];
+		void __iomem *mux[5];
```

Comment:
@merbanan do we really need 2 separated mapping for serdes? do you have any dts example?

Reply:
1 register in the middle.